config/default.yml

```diff
@@ -3065,6 +3065,7 @@ Style/Documentation:
   Description: 'Document classes and non-namespace modules.'
   Enabled: true
   VersionAdded: '0.9'
+  ExcludedConstants: []
```

Comment:
Let's name this `IgnoredConstants` to be consistent with the naming we employ in general (e.g. `IgnoredMethods`).